android/assets/ui/languages.xml

```diff
@@ -2,6 +2,7 @@
     <Language id="en" name="English" fontSet="xolonium-kwajong"/>
     <Language id="fr" name="Français" fontSet="xolonium-kwajong"/>
     <Language id="de" name="Deutsch" fontSet="xolonium-kwajong"/>
+    <Language id="nl" name="Dutch" fontSet="xolonium-kwajong"/>
```

Comment:
I added this entry for Dutch to appear in the UI, but the value of the `name` attribute must be the *translated name* of the language. Can you edit this file to replace "Dutch" here, or tell me how to say Dutch in Dutch :)

Reply:
Dutch in Dutch = **_Nederlands_** 🙂

Reply:
Thanks! Merging this.

Reply:
Thank you for merging! 🙂